tests/ci/sync-to-aws.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -euo pipefail
+
+source /tmp/.env
```

Comment:
Where is the envfile comes from?

Reply:
`./tests/ci/create-commit.sh` and `./tests/ci/install-vm.sh`

Reply:
Those scripts are consuming the envfile, but I can't figure out how it gets to the target environment

Reply:
@lioramilbaum all the scripts from that pipeline needs that file for passing variables. The first one is created at `setup.sh`.
As we split the scripts, we needed a way to pass variables generated in one stage (script) to the next. I wasn't better option at the time.

It's not pretty, but it works and we were able to finish the PoC.

Reply:
can I resolve this? @lioramilbaum